pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -239,10 +239,7 @@ def test_multiindex_groupby_mixed_cols_axis1(func, expected, dtype, result_dtype
         [[1, 2, 3, 4, 5, 6]] * 3,
         columns=MultiIndex.from_product([["a", "b"], ["i", "j", "k"]]),
     ).astype({("a", "j"): dtype, ("b", "j"): dtype})
-    warn = FutureWarning if func == "std" else None
-    msg = "The default value of numeric_only"
-    with tm.assert_produces_warning(warn, match=msg):
-        result = df.groupby(level=1, axis=1).agg(func)
```

Comment:
In 1.5.x we are currently warning in a few erroneous cases with axis=1. Is this worth fixing? I think the PR would have to go straight into 1.5.x because of the enforced deprecations.

Reply:
I would say if we haven't gotten an issue about this yet I think it's okay not fixing for now